src/api/config/feature.yml

```diff
@@ -0,0 +1,2 @@
+features:
+  image_templates: true
```

Comment:
If the file is not there then an exception is thrown by the gem. Something like this:

```
usr/lib64/ruby/gems/2.4.0/gems/feature-1.4.0/lib/feature/repository/yaml_repository.rb:54:in `read': No such file or directory @ rb_sysopen - /vagrant/src/api/config/feature.yml (Errno::ENOENT)
        from /usr/lib64/ruby/gems/2.4.0/gems/feature-1.4.0/lib/feature/repository/yaml_repository.rb:54:in `read_file'
        from /usr/lib64/ruby/gems/2.4.0/gems/feature-1.4.0/lib/feature/repository/yaml_repository.rb:44:in `active_features'
        from /usr/lib64/ruby/gems/2.4.0/gems/feature-1.4.0/lib/feature.rb:56:in `refresh!'
        from /usr/lib64/ruby/gems/2.4.0/gems/feature-1.4.0/lib/feature.rb:121:in `active_features'
        from /usr/lib64/ruby/gems/2.4.0/gems/feature-1.4.0/lib/feature.rb:68:in `active?'
        from /usr/lib64/ruby/gems/2.4.0/gems/feature-1.4.0/lib/feature.rb:87:in `with'
        from /vagrant/src/api/config/routes.rb:67:in `block (2 levels) in <top (required)>'
        from /usr/lib64/ruby/gems/2.4.0/gems/actionpack-5.0.1/lib/action_dispatch/routing/mapper.rb:993:in `block in constraints'
        from /usr/lib64/ruby/gems/2.4.0/gems/actionpack-5.0.1/lib/action_dispatch/routing/mapper.rb:855:in `scope'

```

But I guess that is almost the same as not having database.yml... 😞